driver/nodeserver.go

```diff
@@ -48,7 +47,7 @@ type nodeServer struct {
 func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolumeRequest) (*csi.NodePublishVolumeResponse, error) {
 	meta := metadata.FromNodePublishVolumeRequest(req)
 	log := loggerForMetadata(ns.log, meta)
-	ctx, _ = context.WithTimeout(ctx, time.Second*30)
+	ctx, _ = context.WithTimeout(ctx, time.Second*60)
 
```

Comment:
We should probably be cancelling this context on return.

```suggestion
	ctx, cancel = context.WithTimeout(ctx, time.Second*60)
        defer cancel()
```

```